client/extensions/woocommerce/state/sites/payment-methods/actions.js

```diff
@@ -49,3 +52,45 @@ export const fetchPaymentMethods = ( siteId ) => ( dispatch, getState ) => {
 			dispatch( setError( siteId, getAction, err ) );
 		} );
 };
+
+export const paymentMethodSaveSuccess = ( siteId, data ) => {
```

Comment:
I feel like it is a mistake to inline this into the dispatch.  There is enough going on there that pulling it out into a function makes more sense to me. It definitely does not need to be exported though.